proxy-node-test/src/main/java/uk/gov/ida/notification/helpers/EidasAuthnRequestBuilder.java

```diff
@@ -6,6 +6,7 @@
 import org.opensaml.saml.saml2.core.AuthnContextComparisonTypeEnumeration;
 import org.opensaml.saml.saml2.core.AuthnRequest;
 import org.opensaml.security.credential.Credential;
+import org.opensaml.xmlsec.algorithm.SignatureAlgorithm;
 import org.opensaml.xmlsec.algorithm.descriptors.SignatureRSASHA256;
 import org.opensaml.xmlsec.signature.support.SignatureException;
 import org.opensaml.xmlsec.signature.support.Signer;
```

Comment:
import org.w3c.dom.*;

Reply:
Think the GDS dev guide indicates not to do wildcard import?

Reply:
There is a wild card import on L13 https://github.com/alphagov/verify-proxy-node/blob/f5960ff016b26443f8330c3fce4df024c51202fb/proxy-node-test/src/main/java/uk/gov/ida/notification/helpers/EidasAuthnRequestBuilder.java#L13
couldn't highlight it with the + key

Reply:
Thanks, fixed 👍